--- src/main.c.orig	2010-12-12 09:41:35.738177147 +0000
+++ src/main.c	2010-12-12 09:42:25.726177008 +0000
@@ -123,7 +123,7 @@
         //install_allegro(SYSTEM_AUTODETECT, &errno, atexit);
         
         
-        append_filename(t,exedir,"roms\\tube\\ReCo6502ROM_816",511);
+        append_filename(t,exedir,"../share/b-em/roms/tube/ReCo6502ROM_816",511);
         if (!file_exists(t,FA_ALL,NULL) && selecttube==4) selecttube=-1;
 
         curtube=selecttube;
@@ -268,7 +268,7 @@
         OS01=models[curmodel].os01;
         compactcmos=models[curmodel].compact;
         getcwd(t,511);
-        append_filename(t2,exedir,"roms",511);
+        append_filename(t2,exedir,"../share/b-em/roms",511);
         chdir(t2);
         if (models[curmodel].romsetup) models[curmodel].romsetup();
         loadroms(models[curmodel]);
@@ -343,7 +343,7 @@
         if (models[curmodel].tube!=-1)     curtube=models[curmodel].tube;
         resetmem();
         getcwd(t,511);
-        append_filename(t2,exedir,"roms",511);
+        append_filename(t2,exedir,"../share/b-em/roms",511);
         chdir(t2);
         if (models[curmodel].romsetup) models[curmodel].romsetup();
         loadroms(models[curmodel]);
--- src/mem.c.orig	2010-12-12 10:06:37.475782054 +0000
+++ src/mem.c	2010-12-12 10:07:49.953782019 +0000
@@ -131,7 +131,7 @@
                 fclose(f);
         }
         getcwd(p2,511);
-        sprintf(path,"%sroms/%s",exedir,m.romdir);
+        sprintf(path,"%s../share/b-em/roms/%s",exedir,m.romdir);
         chdir(path);
         loadswroms();
         chdir(p2);
--- src/mem.c.orig	2010-12-12 10:25:59.764781905 +0000
+++ src/mem.c	2010-12-12 10:26:08.642782020 +0000
@@ -104,7 +104,7 @@
         int c=15;
         FILE *f;
         char path[512];
-        sprintf(path,"%sroms/ADFS1-53.rom",exedir);
+        sprintf(path,"%s../share/b-em/roms/ADFS1-53.rom",exedir);
         while (romused[c] && c>=0) c--;
         if (c>=0)
         {
--- src/tapenoise.c.orig	2010-12-12 10:31:01.647781978 +0000
+++ src/tapenoise.c	2010-12-12 10:31:17.218782004 +0000
@@ -23,8 +23,8 @@
         int c;
 
         getcwd(p2,511);
-        if (ddtype) sprintf(path,"%sddnoise",exedir);
-        else        sprintf(path,"%sddnoise",exedir);
+        if (ddtype) sprintf(path,"%s../share/b-em/ddnoise",exedir);
+        else        sprintf(path,"%s../share/b-em/ddnoise",exedir);
 //        printf("path now %s\n",path);
         chdir(path);
         tsamples[0]=load_wav("motoron.wav");
--- src/ddnoise.c.orig	2010-12-12 10:30:53.283782083 +0000
+++ src/ddnoise.c	2010-12-12 10:31:51.087782021 +0000
@@ -23,8 +23,8 @@
 {
         char path[512],p2[512];
         getcwd(p2,511);
-        if (ddtype) sprintf(path,"%sddnoise/35",exedir);
-        else        sprintf(path,"%sddnoise/525",exedir);
+        if (ddtype) sprintf(path,"%s../share/b-em/ddnoise/35",exedir);
+        else        sprintf(path,"%s../share/b-em/ddnoise/525",exedir);
 //        printf("path now %s\n",path);
         chdir(path);
         seeksmp[0][0]=load_wav("stepo.wav");
